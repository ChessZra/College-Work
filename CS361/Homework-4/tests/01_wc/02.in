wc
this is a test
