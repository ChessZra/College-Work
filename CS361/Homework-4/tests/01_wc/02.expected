cs361sh> Num Bytes: 15
Num Lines: 1
Num Words: 4
cs361sh> 
